src/plugins/data_views/common/constants.ts

```diff
@@ -57,3 +57,5 @@ export const PLUGIN_NAME = 'DataViews';
  * @public
  */
 export const FIELDS_FOR_WILDCARD_PATH = '/internal/data_views/_fields_for_wildcard';
+
+export const DATA_VIEWS_FIELDS_FROM_FROZEN = 'data_views:fields_from_frozen';
```

Comment:
just quick feedback, I think the functionality in this PR is already looking good. About the naming, we aim to allow the exclusion of confused data tiers, which can be data_frozen, data_cold....
so something like this could work (of course the logic would need adaptation)

```suggestion
export const DATA_VIEWS_FIELDS_FROM_FROZEN = 'data_views:field_exclude_data_tiers';
```